content/en/community/_index.md

```diff
@@ -65,6 +65,20 @@ channel on [Kubernetes Slack][].
 
 ## Community meetings
 
+etcd contributors and maintainers meet [online][] every week, on **Thursday
+at 11 AM** [Pacific Time][], alternating between community meetings and issue
+triage meetings. Issue triage meetings are aimed at getting through our backlog
+of PRs and Issues. Triage meetings are open to any contributor; you don't have
+to be a reviewer or approver to help out! They can also be a good way to get
+started contributing.
+
+For phone-in information, the date of the next meeting, and minutes from past
+meetings, see [etcd community meeting][meeting-doc].
+
+{{% cards "community_resources" %}}
```

Comment:
Reviewing the deploy preview having two instances of `{{% cards "community_resources" %}}` creates two sections of resources: https://deploy-preview-749--etcd.netlify.app/community

![image](https://github.com/etcd-io/website/assets/39425256/9e734b21-df96-4309-95e3-b6c963a31032)


The problem is that while the Zoom and YouTube resources are the same for both meetings the Triage meetings use a separate Google Doc which is: https://docs.google.com/document/d/15Plser31Dk5kbWVJufQJvHqNhRLQDrBiZ3nY5NPoTts/edit?usp=sharing

We would need to either:

a) Change to using single google doc, just copy the small content we have in separate doc and bring it into main community doc making it clear which meeting is a triage meeting.

b) Introduce a `{{% cards "triage_resources" %}}` include so we can customize resources for triage.

My vote would go to A to keep things simple.  Keen to leave this open for a couple days to see if anyone has any preferences or other suggestions.

Reply:
That makes sense to me - I'll update the PR to go in the "A" direction for others to review